pg/src/main/java/org/spongycastle/bcpg/SignaturePacket.java

```diff
@@ -26,17 +26,17 @@
     private SignatureSubpacket[]   hashedData;
     private SignatureSubpacket[]   unhashedData;
     private byte[]                 signatureEncoding;
-    
+
```

Comment:
Please remove these indent changes from the pull request. Especially when we later want to get this upstream to BouncyCastle, they like to have clean patch sets without unnecessary changes.